PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -356,6 +356,8 @@ def miniAOD_customizeCommon(process):
         updatedTauName = _updatedTauName,
         toKeep = ['deepTau2017v2p1']
     )
+    from Configuration.Eras.Modifier_phase2_common_cff import phase2_common #Phase 2 Tau MVA
+    phase2_common.toModify(tauIdEmbedder, tauIdEmbedder.toKeep.append('newDMwLTwGJPhase2')) #Phase 2 Tau MVA
```

Comment:
> This is nice trick to add Phase 2 Tau MVA with the `runTauIdMVA.py` tool using an era modifier. As it is done like this, I think you can remove L371-408 as `process.rerunIsolationMVADBnewDMwLTPhase2Task` is created by the `runTauIdMVA.py` tool (check if name is same).

If I comment out L371-408 I receive the following error:
```
----- Begin Fatal Exception 02-Jun-2020 10:01:13 CDT-----------------------
An exception of category 'ScheduleExecutionFailure' occurred while
   [0] Calling beginJob
Exception Message:
Unrunnable schedule
Module run order problem found: 
slimmedTaus consumes rerunDiscriminationByIsolationMVADBnewDMwLTPhase2, rerunDiscriminationByIsolationMVADBnewDMwLTPhase2 consumes rerunDiscriminationByIsolationMVADBnewDMwLTPhase2raw, rerunDiscriminationByIsolationMVADBnewDMwLTPhase2raw consumes slimmedTaus
 Running in the threaded framework would lead to indeterminate results.
 Please change order of modules in mentioned Path(s) to avoid inconsistent module ordering.
----- End Fatal Exception -------------------------------------------------
```

Reply:
You should change name of tau input collection of `rerunDiscriminationByIsolationMVADBnewDMwLTPhase2raw` to string stored in `_noUpdatedTauName` similarly to what is done for deepTau in L362 (old version) or L364 (new version). I think it will solve the issue with circular dependency.